packages/synapse-react-client/.storybook/preview.tsx

```diff
@@ -51,57 +51,6 @@ if (process.env.NODE_ENV === 'development') {
   })
 }
 
-export const parameters = {
-  actions: { argTypesRegex: '^on[A-Z].*' },
-  controls: {
-    expanded: true,
-    matchers: {
-      color: /(background|color)$/i,
-      date: /Date$/,
-    },
-  },
-  msw: {
-    handlers: [
-      // Only return mocked data when making requests to our mock stack
-      ...getHandlers(MOCK_REPO_ORIGIN),
-    ],
-  },
-}
-
-export const globalTypes = {
-  showReactQueryDevtools: {
-    name: 'React Query Devtools',
-    defaultValue: false,
```

Comment:
Only logical change was to remove `defaultValue` from `globalTypes`, as it was deprecated.